frontend/src/metabase/lib/dom.js

```diff
@@ -513,3 +513,9 @@ export const getEventTarget = event => {
 
   return target;
 };
```

Comment:
Added this here for now, since I already know I want to reuse this in the other tasks within the epic